src/Data/List/Relation/Unary/Enumerates/Setoid/Properties.agda

```diff
@@ -40,8 +40,8 @@ module _ (S : Setoid a ℓ₁) (T : Setoid b ℓ₂) (surj : Surjection S T) whe
   open Surjection surj
 
   map⁺ : ∀ {xs} → IsEnumeration S xs → IsEnumeration T (map to xs)
-  map⁺ _∈xs y with strictlySurjective y
-  ... | (x , fx≈y) = ∈-resp-≈ T fx≈y (∈-map⁺ S T cong (x ∈xs))
+  map⁺ _∈xs y with (x , fx≈y) ← strictlySurjective y =
```

Comment:
Mostly, when matching on a tuple/record value in this style I *personally* think that the additional parentheses for the sake of (scoping) the pattern, are redundant and noisy... but I see that's a choice, and we don't have `style-guide` strictures to enforce.

Reply:
For some tuples, I do like the parens. But it's not a strongly held belief and I wouldn't fight pushing for such a style. But I'm going to leave this one as is.